src/main/java/algorithms/RegionGrowing.java

```diff
@@ -1,21 +1,127 @@
-package algorithms;
-
-import enums.Metric;
-import enums.ShortName;
-import java.util.Map;
-
-/**
- *
- * @author Jayson
- */
-public class RegionGrowing extends Algorithm {
-
-  public RegionGrowing(ShortName shortName, Map<Metric, Float> weights) {
-    super(shortName, weights);
-  }
-
-  @Override
-  public Boolean start() {
-    return true;
-  }
-}
+package algorithms;
+
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.geom.GeometryCollection;
+import com.vividsolutions.jts.geom.GeometryFactory;
+import com.vividsolutions.jts.io.ParseException;
+import com.vividsolutions.jts.io.WKTReader;
+import com.vividsolutions.jts.io.WKTWriter;
+import enums.Metric;
+import enums.ShortName;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import regions.District;
+import regions.Precinct;
+
+/**
+ *
+ * @author Jayson
+ */
+public class RegionGrowing extends Algorithm {
+
+  List<Precinct> seeds;
+
+  public RegionGrowing(ShortName shortName, Map<Metric, Float> weights) {
+    super(shortName, weights);
+    this.start();
+  }
+
+  @Override
+  public Boolean start() {
+    //JTS configuration 
+    GeometryFactory geometryFactory = new GeometryFactory();
+    WKTReader reader = new WKTReader(geometryFactory);
+    WKTWriter writer = new WKTWriter();
+    //Compactness calculator
+    Schwartzberg szg = new Schwartzberg();
+    
+    //every time add a precinct to the district, newDistrictBoundary grows
+    HashSet<Geometry> newDistrictBoundary = new HashSet<>();
+    //every time add a precinct to the district, also add it's adjPrecncts(ID) to this list as candidates
+    HashSet<Precinct> candidatePrecincts = new HashSet<>();
+
+    //get a seed to start

```

Comment:
A lot of this logic is going to have to get moved around.
Start algorithm should only handle initialization and a call to the first iteration of `run()`.
There's logic in here that needs to be moved to the metrics. Well work on this tomorrow.